content/en/docs/getting-started/_index.md

```diff
@@ -70,9 +70,15 @@ sudo systemctl enable crio --now
 To have `systemd` start and manage MicroShift on Podman, run:
 
 ```Bash
-sudo dnf install -y podman
+sudo dnf install -y podman firewalld
 sudo curl -o /etc/systemd/system/microshift.service \
      https://raw.githubusercontent.com/redhat-et/microshift/main/packaging/systemd/microshift-containerized.service
+sudo systemctl enable firewalld --now
```

Comment:
If I have an existing iptables configuration, for example:

```
# iptables -S
-P INPUT ACCEPT
-P FORWARD ACCEPT
-P OUTPUT ACCEPT
-A INPUT -m state --state RELATED,ESTABLISHED -j ACCEPT
-A INPUT -p icmp -j ACCEPT
-A INPUT -i lo -j ACCEPT
-A INPUT -p tcp -m state --state NEW -m tcp --dport 22 -j ACCEPT
-A INPUT -j REJECT --reject-with icmp-host-prohibited
-A FORWARD -j REJECT --reject-with icmp-host-prohibited
```

And I start `firewalld`:

```
# systemctl enable --now firewalld
```

That wipes out the firewall configuration and I end up with:

```
# iptables -S
-P INPUT ACCEPT
-P FORWARD ACCEPT
-P OUTPUT ACCEPT
```

I don't think we want to be responsible for that. I wonder if we should modify this to say something like:

> Microshift requires the following ports to be open in your firewall configuration in order to functional properly:
>
> ...list of ports here...
>
> If you are already using Firewalld, the following commands will open the necessary ports:
>
> ```
> ...list of firewall-cmd commands here...
> ```
>
> If you are managing your firewall via some other mechanism, you will need to make the appropriate changes to your firewall configuration.

Reply:
@larsks I think it's acceptable compromise, I think we should avoid making the documentation too complicated, we can be opinionated on the method and people shall be smart enough to extrapolate our firewall-cmd commands to their own firewalls?

Just my two cents/pov.